galaxyproject/galaxy
# [24.0] Grid filtering bug fixes

Modified the `Filtering.getValidFilters` function to return an `invalidFilters` object as well, which can be used to tell the user which filters were invalid. Fixes https://github.com/galaxyproject/galaxy/issues/17668

https://github.com/galaxyproject/galaxy/assets/78516064/975e8e3b-9c40-4c4e-819e-a56144e72615

Also added a loading indicator that Fixes https://github.com/galaxyproject/galaxy/issues/17666

## How to test the changes?
(Select all options that apply)
  1. Go to the Workflow List or any modernized grid (Histories, Pages etc.)
  2. Applying any filter not in the list of valid filters (seen in the advanced options) should produce an error and no results.
  3. Tag filtering should work as expected (with and without quotes; producing appropriate results)

## License
